templates/shimexecutor-post-install-job.yaml

```diff
@@ -37,4 +38,58 @@ spec:
                 createDeployment: true
                 deploymentConfig:
                   runtimeClassName: wasmtime-spin-v2
-              EOF
\ No newline at end of file
+              EOF
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}-post-install
+  labels:
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
If we decide that we only want to install the SpinAppExecutor resource into the same namespace as the spinkube helm release, we can/should change this to a namespaced `Role` (and `RoleBinding` below).

Reply:
(updated)